PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,61 @@
+<!--
+BEFORE YOU CREATE A PULL REQUEST:
+
+Ensure you have read over CONTRIBUTING - https://github.com/chocolatey/.github/blob/master/CONTRIBUTING.md. We provide VERY defined guidance (as such, we strongly adhere to it).
+
+A summary of our expectations:
+ - You are not submitting a pull request from your MASTER branch.
+ - You are able to sign the Contributor License Agreement (CLA).
+ - YOUR GIT COMMIT MESSAGE FORMAT IS EXTREMELY IMPORTANT. We have a very defined expectation for this format and are sticklers about it. Really, READ the entire Contributing document. It will save you and us pain.
+ - Do not reformat code, it makes it hard to see what has changed. Leave the formatting to us.
+
+THANKS! We appreciate you reading the entire Contributing document and not just scanning through it.
+
+Name your issue appropriately: give it a sentence that reads well enough for anyone seeing this in release notes to know what the issue is.
+
+When writing out the pull request details please ensure you are writing it as
+if you were explaining it to somebody else.
+Even if you will be working on and resolving the issue yourself.
+This helps others to understand the reasons for the pull request and for it to be searchable in future.
+
+Please do not remove any of the headings.
+If a heading is not applicable then enter N/A: Why it's not applicable
+
+Make sure you have raised an issue for this pull request before continuing.
+
+Please remove all comments before submitting.
+-->
+
+## Description Of Changes
+<!-- Enter a description of the pull request changes -->
+
+## Motivation and Context
+<!-- Why is this change necessary and under what context is it being done -->
+
+## Testing
+<!-- How has this change been tested? If multiple different tests have been done please list them.
+1. Tested this way
+1. Tested that way
+-->
+
+## Change Types Made
+<!-- Tick the boxes for the type of changes that have been made -->
+
+* [ ] Bug fix (non-breaking change)
+* [ ] Feature / Enhancement (non-breaking change)
+* [ ] Breaking change (fix or feature that could cause existing functionality to change)
+
+## Related Issue
+<!-- Make sure you have raised an issue for this pull request before
+continuing. -->
+
+Fixes #
+
+## Change Checklist
+
+* [ ] Requires a change to the documentation
+* [ ] Documentation has been updated
+* [ ] Tests to cover my changes, have been added

```

Comment:
What happens to this on say the docs repository? I don't think this is applicable there.